src/config.ts

```diff
@@ -3,7 +3,7 @@ import { MarkdownPreviewProps } from "@uiw/react-markdown-preview";
 /**
  * The URL for the API (default: https://api.devana.ai/).
  */
-export const API_URL = `https://api.devana.ai/`;
```

Comment:
Issue:
Are we sure we want to use the http version ?
We will need to use https